docs/guides/gke-helm-provider-deployment-guide.md

```diff
@@ -0,0 +1,149 @@
+---
+subcategory: "gke"
+page_title: "GKE Helm Provider Deployment Guide"
+description: |-
+This document walks you through a basic deployment using Twingate's Terraform provider on GKE using the Helm Terraform provider
+---
+
+# Deployment Guide
+
+This deployment guide walks you through a Twingate connector helm deployment in a GKE cluster.
+
+## Before you begin
+
+* Sign up for an account on the [Twingate website](https://www.twingate.com). You will need the Twingate Enterprise tier to use Terraform with Twingate.
+* Create a Twingate [API key](https://docs.twingate.com/docs/api-overview). The key will need to have full permissions to Read, Write, & Provision, in order to deploy Connectors through Terraform.
+
+## Setting up the Provider
+
+First, we need to set up the Twingate Terraform provider by providing your network ID and the API key you provisioned earlier.
+
+```terraform
+provider "twingate" {
+  api_token = "1234567890abcdef"
+  network   = "autoco"
+}
+
+variable "network" {
+  default = "autoco"
+}
+```
+
+
+## Provider requirements
+
+provider versions are excluded from the example below
+
+```terraform
+terraform {
+  required_providers {
+    helm = {
+      source  = "hashicorp/helm"
+    }
+    google = {
+      source  = "hashicorp/google"
+    }
+    twingate = {
+      source  = "twingate/twingate"
+    }
+    docker = {
+      source  = "kreuzwerker/docker"
+    }
+  }
+}
+```
+
+## Provider setup
+
+```terraform
+
+provider "google" {
+  project     = var.project_id
+  region      = var.region
+  zone        = var.zone
+}
```

Comment:
See my comment below, too, but let's create a new section in the guide that defines all the variables that are used with placeholder examples. That allows someone to literally copy & paste as they go without guessing on our intent.